src/widgets/newgrf_debug_widget.h

```diff
@@ -40,6 +40,7 @@ enum SpriteAlignerWidgets {
 	WID_SA_RESET_REL,   ///< Reset relative sprite offset
 	WID_SA_CENTRE,      ///< Toggle centre sprite.
 	WID_SA_CROSSHAIR,   ///< Toggle crosshair.
+	WID_SA_ZOOM,
```

Comment:
Might be useful to mention that this takes `ZOOM_LVL_END` entries, so when the new button gets added the 2x zoom button doesn't "click" the new button.
Maybe even codify it by not putting it at the end of the enumeration, and already adding the right offset for the `WID` after it.